db/seeds.rb

```diff
@@ -5,3 +5,85 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+# (%w[hiking formal flippers sneakers wellies industrial climbing cycling sport].sample)
+
+
+user = User.new(
+  username: "coolguy123",
+  email: "jamn@gmail.com",
+  password: "helloworld"
+)
+puts "saving"
+user.save!
+puts "saved"
+
+
+
+puts "preparing"
+
+shoe = Shoe.new(
+  title: "addiddas new sexy ones!",
+  brand: "nike",
+  size: "8",
+  category: "sneakers",
+  price: "£8",
+  image_url: "https://assets.adidas.com/images/h_840,f_auto,q_auto:sensitive,fl_lossy,c_fill,g_auto/4e894c2b76dd4c8e9013aafc016047af_9366/Superstar_Shoes_White_FV3284_01_standard.jpg",
+  user_id: user.id
+)
+puts "saving"
+shoe.save!
+puts "saved"
+
+
+
+
+user1 = User.new(
+  username: "jimmybob6969",
+  email: "jimmy@testy.com",
+  password: "password"
+)
+puts "saving"
+user1.save!
+puts "saved"
+
+
+
+
+shoe1 = Shoe.new(
+  title: "Proper good hiking boots",
+  brand: "Burtton",
+  size: "10",
+  category: "hiking",
+  price: "£15",
+  image_url: "https://www.grisport.co.uk/images/evolution-hiking-boot-p4594-288665_image.jpg",
+  user_id: user1.id
+)
+puts "saving"
+shoe1.save!
+puts "saved"
+
+shoe2 = Shoe.new(
+  title: "really fancy LV slippers",
+  brand: "Louis Vuitton",
+  size: "6",
+  category: "formal",
+  price: "£20",
+  image_url: "https://uk.louisvuitton.com/images/is/image/lv/1/PP_VP_L/louis-vuitton-dreamy-flat-loafers-shoes--AHDL1PMK54_PM2_Front%20view.jpg",
+  user_id: user1.id
+)
+puts "saving"
+shoe2.save!
+puts "saved"
+
+shoe2 = Shoe.new(
```

Comment:
@oliverbarnes1 is this meant to be shoe 3?

Reply:
it could be I just did it so that the first shoe is "shoe" and the second one is "shoe1" - sorry for the confusion